merge crvusd docs (#5)
CurveDocs/curve-docs: docs/LLAMMA/pegkeeper.md

            assert _new_caller_share <= SHARE_PRECISION  # dev: bad part value

            self.caller_share = _new_caller_share

            log SetNewCallerShare(_new_caller_share)

        ```

    === "Example"

        ```shell
        >>> PegKepper.set_new_caller_share("todo")
        'todo'
        ```

### `withdraw_profit`
!!! description "`PegKeeper.withdraw_profit() -> uint256:`"

    Function to withdraw the profit generated by the PegKeeper.

    Returns: amount of LP tokens (`uint256`).

    Emits: `Profit` event.

    ??? quote "Source code"

        ```python hl_lines="1 6 14 17"
        event Profit:
            lp_amount: uint256

        @external
        @nonpayable
        def withdraw_profit() -> uint256:
            """
            @notice Withdraw profit generated by Peg Keeper
            @return Amount of LP Token received
            """
            lp_amount: uint256 = self._calc_profit()
            POOL.transfer(self.receiver, lp_amount)

            log Profit(lp_amount)

            return lp_amount
        ```

    === "Example"

        ```shell
        >>> PegKepper.withdraw_profit():
        todo
        ```

## **Admin and Receiver**

> PegKeepers have a `admin` and `receiver`. 

Reviewer: ```suggestion
PegKeepers have a `admin` and `receiver` addresses. 
```

> Committing a new admin or receiver requires the corresponding commit functions to be called, actions which can only be performed by the current admin. Subsequently, the newly committed admin or receiver has to apply these changes within a timespan of 3 * 86400 seconds, corresponding to a window of three days. If an attempt is made to apply these changes after this deadline, the action will simply fail.

Reviewer: ```suggestion
Committing a new `admin` or `receiver` requires the corresponding commit functions to be called, actions that can only be performed by the current admin. Subsequently, the newly committed admin or receiver has to apply these changes within a timespan of 3 * 86400 seconds, corresponding to a window of three days. If an attempt is made to apply these changes after this deadline, the action will fail.
```
